extensions/README.md

```diff
@@ -0,0 +1,31 @@
+# stable-diffusion-extensions
+
+## Configuration
```

Comment:
This is cloned by the extensions script that is executed on initialization. No need for review.